pom.xml

```diff
@@ -83,74 +126,57 @@
         </configuration>
       </plugin>
 
-      <!-- This artifact exports the classes whose names are different from the artifactId. -->
       <plugin>
-        <groupId>org.apache.felix</groupId>
-        <artifactId>maven-bundle-plugin</artifactId>
+        <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
Norman and I had discussed possibly moving to gradle, since it would make a lot of this easier ... he's not a fan :)